src/Stripe.net/Infrastructure/Public/StripeConfiguration.cs

```diff
@@ -38,7 +38,7 @@ static StripeConfiguration()
         public static string ApiVersion
         {
             get => apiVersion;
-            set
+            private set
             {
                 apiVersion = value;
                 TrimmedApiVersion = StringUtils.TrimApiVersion(value);
```

Comment:
We should be able to remove TrimmedApiVersion since we are now only allowing users to set the Beta version info

Reply:
I just pushed a commit to the dotnet pr to remove TrimmedApiVersion and also simplify the ApiVersion property (we no longer need to keep a separate variable for storage)